src/MyComponent.tsx

```diff
@@ -0,0 +1,110 @@
+            <img
+              loading="lazy"
+              src="https://roadmap-beta.vercel.app/static/media/logoDark.7ced066b149879ee7973c36223af2885.svg"
+              className="shrink-0 max-w-full aspect-[5.56] w-[209px]"
+              alt="dark_concepts"
+            />
+          }
+          <div className="flex gap-5 items-start py-1.5">
+            <div className="flex gap-1.5 items-start pt-1.5 pb-1 text-base font-bold leading-5 whitespace-nowrap text-neutral-800">
+              <div>EN</div>
+              <img
+                loading="lazy"
+                src="https://cdn.builder.io/api/v1/image/assets/TEMP/982d412dfad3b7e40a307f3fae3acbaa4fd1afa827972129a680a93eca08e2e8?"
+                className="shrink-0 mt-1.5 w-3 aspect-[1.72]"
+                alt="caret"
+              />
+            </div>
+            <div className="flex justify-center items-center py-1.5 mt-1.5 bg-zinc-400">
+              <img
+                loading="lazy"
+                src="https://cdn.builder.io/api/v1/image/assets/TEMP/4f7094a65e92a67234cd3f11c943c14a96506ef574accb6a89beba9cffeaf78a?"
+                className="w-px aspect-[0.1]"
+                alt="seperater_first"
+              />
+            </div>
+            <div className="pt-1.5 self-stretch text-base leading-5 text-center text-zinc-600">
+              Download
+            </div>
+            <div className="pt-1.5 self-stretch text-base leading-5 text-center text-zinc-600">
+              Learn
+            </div>
+            <div className="flex justify-center items-center py-1.5 mt-1.5 bg-zinc-400">
+              <img
+                loading="lazy"
+                src="https://cdn.builder.io/api/v1/image/assets/TEMP/f415dd458764b812e28a480afe022843ad49cf4979fbe19d584720995139db1b?"
+                className="w-px aspect-[0.1]"
+                alt="seperater_second"
+              />
+            </div>
+            <img
+              loading="lazy"
+              src="https://cdn.builder.io/api/v1/image/assets/TEMP/aeb465313cae6a0ff7ff69df7c6bc99f23391d0f20bb5379e11b799d03cd4171?"
+              className="shrink-0 mt-2 aspect-[0.93] w-[15px]"
+              alt="magnifier"
+            />
+          </div>
+        </div>
+        <Roadmap />
+        <RoadmapContent />
+        <MobileSelection />
+        <div className="flex gap-0 mt-8 text-base leading-5 max-md:flex-wrap">
+          <img
+            loading="lazy"
+            src="https://cdn.builder.io/api/v1/image/assets/TEMP/b1fd7cbd65083269ea8f4ac8a66cc0eb7c2cb5b4d24860622286cd5f611dd5d0?"
+            className="shrink-2 aspect-[0.04] w-[37px]"
+            style={{ height: imageHeight }}
+            alt="line"
+          />
+          <div className="grow shrink-0 pl-8 basis-0 w-fit max-md:max-w-full">
+            <div id="parentDiv" className="flex flex-col " ref={contentRef}>
+              <WorkinProgress />
+              <BrandNew />
+              <PastReleases />
```

Comment:
Should use the same class object with different props.
It breaks the reusability.
For example, if you implement the component like this, you should create 20 more classes if I add 20 more list items to the ui.